muxers/yamux/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 - Update to `libp2p-core` `v0.39.0`.
 
+# 0.43.0-alpha
+
+- Update to `yamux` `v0.11.0`.
+  This version contains a major refactoring which hopefully does not break anything.
+  To express this uncertainty, this is released as an `alpha`.
```

Comment:
I don't think we will see any testing of the `alpha`, thus I suggest not doing the alpha release in the first place, but instead release as a new minor version right away.

Reply:
We did release the alpha already :D